cmd/sf/sf.go

```diff
@@ -491,7 +492,13 @@ func main() {
 			ctxts <- ctx
 			identifyRdr(os.Stdin, ctx, ctxts, getCtx)
 		} else {
-			err = identify(ctxts, v, "", *coe, *nr, d, getCtx)
+			globs, err := filepath.Glob(v)
+			if err != nil {
+				log.Fatalln(fmt.Sprintf("[FATAL] bad glob pattern: %e", err))
```

Comment:
Sorry @richardlehane `log.Fatalln(fmt.Sprintf("[FATAL] bad glob pattern: %e", err))` should be `log.Fatalln(fmt.Sprintf("[FATAL] bad glob pattern: %s", err))` -- `%e` will return something like `%!e(string=someError)`. My bad!